src/Core/UI/Components/Controls/TextBox/InputType.cs

```diff
@@ -21,6 +22,7 @@ public static string GetInputType(this InputType inputType)
             InputType.Number => "number",
             InputType.Tel => "tel",
             InputType.Currency => "currency",
+            InputType.Percentage => "percentage",
```

Comment:
Isso é um bug não apenas da sua PR mas que acabei pegando vendo esse método:

![image](https://github.com/user-attachments/assets/9bca0b94-eb26-4bdc-9f9f-9ade2ccbb0c9)

Qndo for percentage e currency, pode retornar como "number"